submissions/okkkko/html_css_popup/index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
```

Comment:
According to HTML5 specification all single tags should be without slash "/". Could you please remove all slashes?